interfaces/builtin/x11.go

```diff
@@ -138,6 +139,13 @@ owner /run/user/[0-9]*/mutter/Xauthority r,
 network netlink raw,
 /run/udev/data/c13:[0-9]* r,
 /run/udev/data/+input:* r,
+
+# Deny access to ICE granted by abstractions/X
+# See: https://bugs.launchpad.net/snapd/+bug/1901489
+deny owner @{HOME}/.ICEauthority r,
```

Comment:
Would it make sense to also deny access to `/run/user/[0-9]*/ICEauthority`?  It isn't currently granted by `<abstractions/X>`, but I could see that being changed in a future version of AppArmor.

Reply:
This sounds like a good idea to make it future-proof.

Reply:
I agree, I'll make this change shortly.

Reply:
Done